### ISSUE TITLE
Install recommendation notification does not resolve publisher name

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#233640]
IIRC this used to say Microsoft instead of ms-vscode

![Image](https://github.com/user-attachments/assets/b0dfd334-e8be-46bf-9ac6-63308d3e6b57)

**Comment by sandy081** [comment]
Thats right, I believe this is coming from the workspace extension. Can you please check if you have a workspace extension under `.vscode/extensions` folder with publisher `ms-vscode` ? If so, we cannot resolve the publisher of the workspace extension.

**Comment by joyceerhl** [comment]
Not sure which one of these it is: https://github.com/microsoft/vscode/blob/b0f63370b3bb9e44e5510068142d33203b6f129e/.vscode/extensions.json#L1-L13

**Comment by sandy081** [comment]
Do you have a folder `extensions` under `.vscode` folder? If so what extensions does it have?

**Comment by joyceerhl** [comment]
No, this is just the VS Code repo

**Comment by sandy081** [comment]
VS Code repo has workspace extensions (available under .vscode/extensions folder), I believe above prompt is for these workspace extensions. Probably I should not show publisher information for workspace extensions.
